index.html

```diff
@@ -0,0 +1,160 @@
+            <div class="demo">
+                <h6> @ the pensula staduim hrq</h6>
+            </div>
+        </div>
+
+    </section>
+    <div class="section2">
+        <h1> About Program</h1>
+        <div class="guide-bar"></div>
+    </div>
+
+    <div class="section3">
+        <h2 class="sectionh1"> main speaker</h2>
+        <div class="guided-bar"></div>
+    </div>
+
+    <div class="section4">
+    </div>
+    <section class="partners d-lg-none">
+        <div class="container-fluid">
+            <div class="section-title">
+                <h4>Partner</h4>
+                <hr>
+            </div>
+            <div class="partners-info">
+                <ul class="list-unstyled">
+                    <li><a href="https://www.amazon.com/" target="_blank"><i class="fab fa-amazon fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://www.apple.com/" target="_blank"><i class="fab fa-apple fa-3x"></i></a></li>
+                    <li><a href="https://about.fb.com/company-info/" target="_blank">
+                            <i class="fab fa-facebook-square fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://www.paypal.com/" target="_blank"><i class="fab fa-cc-paypal fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://www.spotify.com/" target="_blank"><i class="fab fa-spotify fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://www.twitch.tv/" target="_blank"><i class="fab fa-twitch fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://usa.visa.com/" target="_blank"><i class="fab fa-cc-visa fa-3x"></i></a>
+                    </li>
+                    <li><a href="https://www.xbox.com/" target="_blank"><i class="fab fa-xbox fa-3x"></i></a></li>
+                    <li><a href="https://www.youtube.com/about/" target="_blank"><i
+                                class="fab fa-youtube fa-3x"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <footer class=" footer">
+
+        <div class="container-fluid">
+            <div class="logo mr-5 mr-sm-3">
+                <img src="/assets/img/loachella-logo-third.svg" alt="logo-footer">
+            </div>
+            <div class="info-footer">
+                <div class="titulo d-lg-flex">
+                    <h6 class="font-weight-bold mr-lg-1">2020 Loachella Group </h6>
+                    <h6 class="font-weight-bold"> All rights reserved </h6>
+                </div>
+                <p class="d-none d-lg-block">Welcome to AEG’s Privacy Request Portal. This form allows you to submit
+                    certain data rights requests across the AEG family of companies. </p>
+            </div>
+        </div>
+
+    </footer>
+    <script src="js/menulis.js"></script>
+    <script src="js/aboutProgram.js"></script>
+    <script src="/js/speakerdata.js"></script>
```

Comment:
- [ ] Please import your js files correct as they should be imported like that
`<script src="./js/menulis.js"></script>`
`<script src="./js/aboutprogram.js"></script>`
`<script src="./js/speakerdata.js"></script>`